dashboards/log-analysis/log-analysis.json

```diff
@@ -420,7 +420,7 @@
             "nrqlQueries": [
               {
                 "accountId": 0,
-                "query": "SELECT count(*) FROM Log WHERE trace.id IS NOT NULL since 1 hour ago facet entity.name limit 100"
+                "query": "SELECT count(*) FROM Log WHERE entity.guid IS NOT NULL since 1 hour ago facet entity.name limit max"
```

Comment:
@JimHagan - Since we're filtering on `entity.guid`, this may not work for another user's account if they attempt to install this dashboard. Its a small caveat, however we would need to have a more specific variable to replace `entity.guid`.

Reply:
@josephgregoryii 

We are only checking to see if the entity.guid is NULL not checking the absolute value of the GUID.  We know that if there is an entity.guid we can facet by entity name.

Reply:
Sounds good to me!